components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-overlay {
+  /* TODO: replace with core tokens */
+  --spectrum-overlay-animation-distance: 6px;
+  --spectrum-global-animation-duration-100: 130ms;
```

Comment:
Can you use the `--spectrum-animation-duration-100` in `custom-vars.css`?

You could also define a token with the overlay prefix here e.g. `--spectrum-overlay-animation-duration: var(--spectrum-animation-duration-100);`.

Reply:
Good catch, resolved this in a new commit.